ui/core-react/src/core-react/icons/IconComponent.tsx

```diff
@@ -49,10 +55,17 @@ export function Icon(props: IconProps) {
       : undefined;
 
   if (iconString) {
-    const webComponentString =
-      IconSpecUtilities.getWebComponentSource(iconString);
-    if (webComponentString) {
-      const svgLoader = `<svg-loader src="${webComponentString}"></svg-loader>`;
+    const webComponentString = getWebComponentSource(iconString);
+
+    if (
+      iconString.startsWith("data:") ||
```

Comment:
Since we are adding "allowed" values to `iconSpec`, can you add in the `IconSpec` type above the description of what is supported for the strings ?
IE:
```ts
/** Prototype for an IconSpec which can be a string, ReactNode or ConditionalStringValue.
 * Strings are expected to be the following:
 *  - Created by AppUi Abstract `IconSpecUtilities.createWebComponentIconSpec` (deprecated)
 *  - A URL to a SVG file (must end with ".svg").
 *  - A dataURI with MimeTypes `image/svg+xml` (ensure valid dataURI encoding) or `image/svg+xml;base64`;
 *  - Anything else will be treated as a class name and be directly added to the `class` attribute.
 * @public
 */
export type IconSpec =
  | string
  | ConditionalStringValue
  | React.ReactNode
  | ConditionalIconItem;
```

Reply:
This is done despite GH not showing the update here.